docs/site/docusaurus.config.js

```diff
@@ -108,6 +108,214 @@ const config = {
         intentionallyNotExported: [],
       },
     ],
+    [
```

Comment:
This should be done with a wildcard, no?

Reply:
Yes, why not? 
Will update in the next commit. 🙌